packages/dds/test-dds-utils/src/ddsFuzzHarness.ts

```diff
@@ -1588,6 +1589,35 @@ export async function replayTest<
 	await runTestForSeed(model, options, seed, saveInfo);
 }
 
+export function generateTestSeeds(testCount: number, stressMode: StressMode): number[] {
+	const random = makeRandom();
+
+	const seeds = Array.from({ length: testCount }, (_, i) => i);
+
+	switch (stressMode) {
+		case StressMode.Short: {
```

Comment:
we should just pick a random initial seed and use consecutive seeds from there.

this strategy just runs the tests in a random order, but still uses the same seeds. it also doesn't do so very performantly

Reply:
this still seems to apply in latest. In general it doesn't make sense to limit us to use seeds 0 through `testCount` for the random scenarios, and there's definitely no need to run the same seed more than once as is currently done in `Long`.

Instead you should be doing something like picking the start number between 0 and max_safe_int for Long and just using the seeds between `start` and `start + testCount`. No need for further randomness (running tests in a different order doesn't do anything interesting)